layouts/default.vue

```diff
@@ -1,9 +1,32 @@
 <template>
   <div>
+
+    <div class="hidden md:block">
+      <navBar/>
```

Comment:
@jacquesme 
All .vue components should be named and exported using Uppercase. E.g.```NavBar```.
Then you'll always see where are vue component and where are regular HTML tags in vue templates.